shell/src/main/java/alluxio/cli/fsadmin/journal/QuorumElectCommand.java

```diff
@@ -60,11 +90,92 @@ public static String description() {
+    List<InetSocketAddress> addresses = mRaftJournalConf.getClusterAddresses();
+    Set<RaftPeer> peers = addresses.stream()
+        .map(addr -> RaftPeer.newBuilder()
+            .setId(RaftJournalUtils.getPeerId(addr))
+            .setAddress(addr)
+            .build()
+        )
+        .collect(Collectors.toSet());
+    mRaftGroup = RaftGroup.valueOf(RAFT_GROUP_ID, peers);
+
+    String strAddr = cl.getOptionValue(ADDRESS_OPTION_NAME);
+    NetAddress address = QuorumCommand.stringToAddress(strAddr);
+    InetSocketAddress serverAddress = InetSocketAddress
+        .createUnresolved(address.getHost(), address.getRpcPort());
+    List<RaftPeer> oldPeers = new ArrayList<>(mRaftGroup.getPeers());
+    // if you cannot find the address in the quorum, throw exception.
+    if (oldPeers.stream().map(RaftPeer::getAddress).noneMatch(addr -> addr.equals(strAddr))) {
+      throw new IOException(String.format("<%s> is not part of the quorum <%s>.",
+          strAddr, oldPeers.stream().map(RaftPeer::getAddress).collect(Collectors.toList())));
+    }
+
+    RaftPeerId newLeaderPeerId =
+        RaftJournalUtils.getPeerId(address.getHost(), address.getRpcPort());
+    /* update priorities to enable transfer */
+    List<RaftPeer> peersWithNewPriorities = new ArrayList<>();
+    for (RaftPeer peer : oldPeers) {
+      peersWithNewPriorities.add(
+          RaftPeer.newBuilder(peer)
+              .setPriority(peer.getId().equals(newLeaderPeerId) ? 2 : 1)
+              .build()
+      );
+    }
+    try (RaftClient client = createClient()) {
+      String stringPeers = "[" + peersWithNewPriorities.stream().map(RaftPeer::toString)
+          .collect(Collectors.joining(", ")) + "]";
+      mPrintStream.printf(
+          "Applying new peer state before transferring leadership: %s%n", stringPeers);
+      RaftClientReply reply = client.admin().setConfiguration(peersWithNewPriorities);
+      processReply(reply, "failed to set master priorities before initiating election");
+      /* transfer leadership */
+      mPrintStream.printf(
+          "Transferring leadership to master with address <%s> and with RaftPeerId <%s>%n",
+          serverAddress, newLeaderPeerId);
+      // fire and forget: need to immediately return as the master will shut down its RPC servers
+      // once the TransferLeadershipRequest is initiated.
+      final int SLEEP_TIME_MS = 3_000;
+      final int TRANSFER_LEADER_WAIT_MS = 60_000;
+      try {
+        Thread.sleep(SLEEP_TIME_MS);
+        RaftClientReply reply1 = client.admin().transferLeadership(newLeaderPeerId,
+            TRANSFER_LEADER_WAIT_MS);
+        processReply(reply1, "election failed");
+      } catch (Throwable t) {
+        mPrintStream.printf("caught an error when executing transfer: %s%n", t.getMessage());
+        return -1;
+      }
+      mPrintStream.println("Transferring leadership initiated");
+    } catch (Throwable t) {
+      throw new IOException(t);
```

Comment:
Why catch the `Throwable` in the above block but throw here?

Reply:
No need for this place, thanks for point it out, have remove this useless catch block